src/Token/AccessToken.php

```diff
@@ -84,6 +89,17 @@ public function __construct(array $options = [])
 
             $this->expires = $expires;
         }
+
+        // Capure any additional values that might exist in the token but are
+        // not part of the standard response. Vendors will sometimes pass
+        // additional user data this way.
+        $this->values = array_diff_key($options, array_flip([
+            'access_token',
+            'resource_owner_id',
+            'refresh_token',
+            'expires_in',
+            'expires',
+        ]));
```

Comment:
Maybe eventually. For now all the keys are in the same function so it seems fine to me.